Automattic/woocommerce-payments
# Remove Remind Me Later option from the FRT discoverability banner

Fixes #6321 

#### Changes proposed in this Pull Request





This PR removes the Remind Me Later option from the Fraud and Risk Tools discoverability banner, leaving only the 'Learn more' and 'Dismiss' buttons.





#### Testing instructions





* Check out this branch.
* Build the client: `npm run build:client`
* All tests should pass.
* Delete the old banner settings option from DB: `npm run wp option delete wcpay_frt_discover_banner_settings`
* If required delete the new option that this PR introduces (if you tested previously): `npm run wp option delete wcpay_frt_discover_banner_dismissed`

**Laptop test**

* Navigate to the WCPay Overview page: `/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Foverview`
* You should see the FRT discoverability banner displayed at the top of the page with a 'Learn more' button and a 'Dismiss' button only.

![Screenshot on 2023-07-04 at 11_20_31](https://github.com/Automattic/woocommerce-payments/assets/60988591/9df1ab07-d015-411a-8ce7-7b71a0c6dcaa)

* Check the banner settings option in the DB: `npm run wp option get wcpay_frt_discover_banner_dismissed` 
  * It should return an empty option value
* Navigate to any other page in the admin, after that page loads, navigate back to the WCPay Overview page. The banner should display as it did before. 
* Click the 'Learn more' button. It should direct you to the WCPay settings page as it did before.
* Navigate back to the WCPay Overview page and click the `Dismiss` button. The banner should disappear.
* Check the banner settings option in the DB: `npm run wp option get wcpay_frt_discover_banner_dismissed` 
  * It should return: `1`
* Navigate to any other page in the admin, after that page loads, navigate back to the WCPay Overview page. The banner should ***not*** render.

**Smartphone test**

* Activate Jurassic Tube for your local site: `npm run tube:start`.
* Delete banner settings option from DB: `npm run wp option delete wcpay_frt_discover_banner_settings`
* Visit your local site in your smartphone and login.
* Navigate to the WCPay Overview page: `/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Foverview`
* You should see the FRT discoverability banner displayed at the top of the page with a 'Learn more' button and a 'Dismiss' button only.
* Check the banner settings option in the DB: `npm run wp option get wcpay_frt_discover_banner_dismissed` 
  * It should return an empty option value
* Navigate to any other page in the admin, after that page loads, navigate back to the WCPay Overview page. The banner should display as it did before. 
* Click the 'Learn more' button. It should direct you to the WCPay settings page as it did before.
* Navigate back to the WCPay Overview page and click the `Dismiss` button. The banner should disappear.
* Check the banner settings option in the DB: `npm run wp option get wcpay_frt_discover_banner_dismissed` 
  * It should return: `1`
* Navigate to any other page in the admin, after that page loads, navigate back to the WCPay Overview page. The banner should ***not*** render.

-------------------


**Post merge**
